src/Kunstmaan/PagePartBundle/Twig/Extension/PageTemplateTwigExtension.php

```diff
@@ -72,7 +72,7 @@ public function renderPageTemplate(array $twigContext, HasPageTemplateInterface
 
         $template = $this->environment->loadTemplate($pageTemplate->getTemplate());
 
-        return $template->render($twigContext);
+        return $template->render(array_merge($twigContext, $parameters));
```

Comment:
It's better to put $parameters as the first element. The keys from the first array will be preserved. If an array key exists in both arrays, then the element from the first array will be used and the matching key's element from the second array will be ignored.